docs/metrics-events.md

```diff
@@ -67,6 +67,8 @@ in a sign-in or sign-up flow:
 |`password.forgot.resend_code.completed`|A password reset email has been re-sent to the user.|
 |`password.forgot.verify_code.start`|A user has clicked on the link in a password reset email.|
 |`password.forgot.verify_code.completed`|A password reset has been successfully completed on the server.|
+|`route.${path}.200`| A route responded with a 200 status code. Example: `route./account/login.200`|
```

Comment:
I initially suggested we replace the `/` with `.`, but then after building the signup funnel, I felt seeing the `/` would make it easier to recognize which route was reported. And staying slashes, we could build queries like `WHERE type LIKE 'route.%.4%` to grab requests that had 4XX responses (I think, I don't recall if MySQL's `%` is too greedy).

But if people have other objective reasons, I can switch!